### ISSUE TITLE
Documents created by inflator miss public_trial

### ISSUE THREAD

**Issue by deiferni** [4teamwork/opengever.core#1918]
Set up a new gever deployment, open a document overview, 💥 :

```
   - URL: /Users/deif/Files/workspaces/opengever.core/opengever/document/browser/templates/public_trial.pt
   - Line 2, Column 4
   - Expression: <PathExpr standard:u'view/context/public_trial'>
   - Names:
      {'args': (<opengever.document.browser.overview.Overview object at 0x117b5f650>,),
       'context': <Document at /fd/ordnungssystem/fuehrung/kommunikation/allgemeines/dossier-1/document-1>,
       'default': <object object at 0x100297c50>,
       'loop': {},
       'nothing': None,
       'options': {},
       'repeat': {},
       'request': <HTTPRequest, URL=http://localhost:8080/fd/ordnungssystem/fuehrung/kommunikation/allgemeines/dossier-1/document-1/tabbed_view/listing>,
       'template': <zope.browserpage.viewpagetemplatefile.ViewPageTemplateFile object at 0x10f6ec8d0>,
       'view': <opengever.document.browser.overview.Overview object at 0x117b5f650>,
       'views': <zope.browserpage.viewpagetemplatefile.ViewMapper object at 0x117537d10>}
  Module Products.PTProfiler.ProfilerPatch, line 35, in __patched_call__
  Module zope.tales.expressions, line 217, in __call__
  Module zope.tales.expressions, line 194, in _eval
  Module zope.tales.expressions, line 124, in _eval
  Module zope.pagetemplate.engine, line 66, in __call__
  Module zope.traversing.adapters, line 136, in traversePathElement
   - __traceback_info__: (<Document at /fd/ordnungssystem/fuehrung/kommunikation/allgemeines/dossier-1/document-1>, 'public_trial')
  Module zope.traversing.adapters, line 50, in traverse
   - __traceback_info__: (<Document at /fd/ordnungssystem/fuehrung/kommunikation/allgemeines/dossier-1/document-1>, 'public_trial', [])
LocationError: (<Document at /fd/ordnungssystem/fuehrung/kommunikation/allgemeines/dossier-1/document-1>, 'public_trial')
```

or document listing:

```
   - URL: /Users/deif/Files/workspaces/opengever.core/opengever/document/browser/templates/public_trial.pt
   - Line 2, Column 4
   - Expression: <PathExpr standard:u'view/context/public_trial'>
   - Names:
      {'args': (<opengever.document.browser.overview.Overview object at 0x115e21090>,),
       'context': <Document at /fd/ordnungssystem/fuehrung/kommunikation/allgemeines/dossier-1/document-1>,
       'default': <object object at 0x100297c50>,
       'loop': {},
       'nothing': None,
       'options': {},
       'repeat': {},
       'request': <HTTPRequest, URL=http://localhost:8080/fd/ordnungssystem/fuehrung/kommunikation/allgemeines/dossier-1/document-1/tabbed_view/listing>,
       'template': <zope.browserpage.viewpagetemplatefile.ViewPageTemplateFile object at 0x10f553490>,
       'view': <opengever.document.browser.overview.Overview object at 0x115e21090>,
       'views': <zope.browserpage.viewpagetemplatefile.ViewMapper object at 0x115c67390>}
  Module Products.PTProfiler.ProfilerPatch, line 35, in __patched_call__
  Module zope.tales.expressions, line 217, in __call__
  Module zope.tales.expressions, line 194, in _eval
  Module zope.tales.expressions, line 124, in _eval
  Module zope.pagetemplate.engine, line 66, in __call__
  Module zope.traversing.adapters, line 136, in traversePathElement
   - __traceback_info__: (<Document at /fd/ordnungssystem/fuehrung/kommunikation/allgemeines/dossier-1/document-1>, 'public_trial')
  Module zope.traversing.adapters, line 50, in traverse
   - __traceback_info__: (<Document at /fd/ordnungssystem/fuehrung/kommunikation/allgemeines/dossier-1/document-1>, 'public_trial', [])
LocationError: (<Document at /fd/ordnungssystem/fuehrung/kommunikation/allgemeines/dossier-1/document-1>, 'public_trial')
2016-06-13 16:17:40 ERROR Zope.SiteErrorLog 1465827460.140.710137741436 http://localhost:8080/fd/ordnungssystem/fuehrung/kommunikation/allgemeines/dossier-1/@@tabbed_view/listing
Traceback (innermost last):
  Module ZPublisher.Publish, line 138, in publish
  Module ZPublisher.mapply, line 77, in mapply
  Module ZPublisher.Publish, line 48, in call_object
  Module ftw.tabbedview.browser.tabbed, line 160, in listing
  Module opengever.tabbedview.browser.tabs, line 148, in __call__
  Module ftw.tabbedview.browser.listing, line 124, in __call__
  Module ftw.tabbedview.browser.listing, line 274, in render_listing
  Module ftw.table.utils, line 102, in generate
  Module ftw.table.utils, line 212, in get_value
  Module opengever.tabbedview.browser.tabs, line 40, in translate_public_trial_options
  Module Products.CMFCore.FSPythonScript, line 127, in __call__
  Module Shared.DC.Scripts.Bindings, line 322, in __call__
  Module Shared.DC.Scripts.Bindings, line 359, in _bindAndExec
  Module Products.PythonScripts.PythonScript, line 344, in _exec
  Module script, line 21, in translate
   - <FSPythonScript at /fd/translate>
   - Line 21
  Module Products.CMFPlone.TranslationServiceTool, line 49, in translate
  Module zope.i18n, line 126, in translate
  Module zope.i18n.translationdomain, line 78, in translate
  Module zope.i18n.translationdomain, line 129, in _recursive_translate
  Module Products.PlacelessTranslationService.lazycatalog, line 32, in queryMessage
  Module gettext, line 403, in ugettext
TypeError: unhashable type: 'Missing.Value'
```

**Comment by lukasgraf** [comment]
Not fixed yet - while the indexer was definitely an issue, there's still a problem with attribute access for the `public_trial` field.

**Comment by phgross** [comment]
@lukasgraf wurde dieser Issue mit #1952 behoben oder sind hierzu noch Arbeiten offen?

**Comment by lukasgraf** [comment]
Fixed via #1952
